content/guides/additional/remote-scry.md

```diff
@@ -10,6 +10,21 @@ referentially transparent namespace. In other words, it's a function from a
 constrained). Previously we only supported scrying within the same ship, but
 from Kernel version `[%zuse 413]`, it is possible to scry from *other* ships.
 
+{% callout %}
+
+**Warning**
+
+Remote scries are still a new, experimental feature.
+
+As of `[%zuse 413]`, there is a known bug that causes subscribers to often
+fail. A fix is currently planned for `[%zuse 412]`, and it is not recommended
+to use remote scries in production userspace code until then.
```

Comment:
The fix for this goes out today and does not require an additional kelvin burn. After today's release goes out, this message could be updated to reference the base hash/urbit-os version that has the fix.

Reply:
Where is this used in Clay? Would be great to include a reference to the code that can be used to implement a fallback.